scaleapi/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.14.4"
+__version__ = "2.14.5"
```

Comment:
I think you can even make it 2.15.0  this looks like a relatively major improvement to the SDK!

Reply:
Sounds good, I've updated the version to 2.15.0